journals/app.json

```diff
@@ -0,0 +1,30 @@
+{
+  "expo": {
+    "name": "journals",
+    "slug": "journals",
+    "version": "1.0.0",
+    "orientation": "portrait",
+    "icon": "./assets/icon.png",
+    "userInterfaceStyle": "light",
+    "splash": {
+      "image": "./assets/splash.png",
+      "resizeMode": "contain",
+      "backgroundColor": "#ffffff"
+    },
+    "assetBundlePatterns": [
+      "**/*"
+    ],
+    "ios": {
+      "supportsTablet": true
+    },
+    "android": {
+      "adaptiveIcon": {
+        "foregroundImage": "./assets/adaptive-icon.png",
+        "backgroundColor": "#ffffff"
+      }
+    },
+    "web": {
+      "favicon": "./assets/favicon.png"
+    }
+  }
+}
```

Comment:
Is it possible to complete the necessary setup of this example using npm? If so, you should test the install flow on a clean machine (if possible) and write up the steps in a README. If this is not possible, the configuration files and assets should probably be removed.